debian/changelog

```diff
@@ -1,8 +1,10 @@
 auto-upgrade-testing (0.3.1) UNRELEASED; urgency=medium
 
   * debian/control: Add a dependency on python3-junitparser.
+  * d/tests: replace broken `upgrade` test with simple `--help` call
+  * d/gbp.conf: dch: don't merge commits by maintainer (multimaint = false)
 
- -- Brian Murray <brian@ubuntu.com>  Wed, 14 Sep 2022 17:01:26 -0700
+ -- Paride Legovini <paride@ubuntu.com>  Thu, 15 Sep 2022 13:09:13 +0200
```

Comment:
I'm not fussed about having my name in the changelog but is there a reason you didn't use `dch -a`?